waggle-dance-core/src/main/java/com/hotels/bdp/waggledance/client/ThriftMetastoreClientManager.java

```diff
@@ -49,24 +49,26 @@
   private static final Logger LOG = LoggerFactory.getLogger(ThriftMetastoreClientManager.class);
 
   private static final AtomicInteger CONN_COUNT = new AtomicInteger(0);
-
+  private final HiveConf conf;
+  private final HiveCompatibleThriftHiveMetastoreIfaceFactory hiveCompatibleThriftHiveMetastoreIfaceFactory;
+  private final URI[] metastoreUris;
   private ThriftHiveMetastore.Iface client = null;
   private TTransport transport = null;
   private boolean isConnected = false;
-  private URI[] metastoreUris;
   private String tokenStrForm;
-  private final HiveConf conf;
-  private final HiveCompatibleThriftHiveMetastoreIfaceFactory hiveCompatibleThriftHiveMetastoreIfaceFactory;
-
   // for thrift connects
   private int retries = 5;
   private long retryDelaySeconds = 0;
 
-  public ThriftMetastoreClientManager(
+  private final int connectionTimeout;
+
+  ThriftMetastoreClientManager(
```

Comment:
why did you remove public from this class, this is technically a backward incompatible change (it's probably fine) just wondering why

Reply:
^ I guess it's sorted with the version number. (reading some comments above)

Reply:
I was just going to comment to this; it was an IntelliJ change and yeah, it should be covered by the minor version change. However, this is not the aim of this PR, so I'm tempted to put it back to `public`...

Reply:
it's ok